app/javascript/services/fetchAudits.jsx

```diff
@@ -8,13 +8,7 @@ const fetchAudits = async (filters) => {
    */
   const buildFilter = (filters) => {
     let filterString =
-      "?class_name=" +
-      filters.className +
-      "&audit_action=" +
-      filters.auditAction +
-      "&instance_ids=[" +
-      filters.instanceIds +
-      "]" +
+      `?class_name=${filters.className}&audit_action=${filters.auditAction}&instance_ids=[${filters.instanceIds}]` +
```

Comment:
I meant the `query-string` module (since it's already among the dependencies), like this:

```js
const query = {
  class_name: filters.className,
  audit_action: filters.auditAction,
  instance_ids: filters.instanceIds,
};

if (filters.dateFrom) {
  query.date_from = filters.dateFrom;
}

return queryString.stringify(query, { arrayFormat: "comma" });
```

I'd also recommend using something like the `humps` module to simplify the key conversion:

```js
const query = humps.decamelize(filter);
return queryString.stringify(query, { arrayFormat: "comma" });
```